src/react-apps/applications/dashboard/src/App.tsx

```diff
@@ -59,7 +59,7 @@ class App extends React.Component<IDashboardProps, IMainDashboardState> {
         <Router>
           <div>
             <AppBarComponent
-              org={this.props.user.full_name || this.props.user.login}
+              org={this.props ? this.props.user.full_name || this.props.user.login : ''}
```

Comment:
Do you need to check this.props before this.props.user.full_name?

Reply:
The check was going to be for this.props.user, fixed